src/class/api.ga4gh.php

```diff
@@ -862,10 +932,9 @@ private function showVariantDataPage ($sBuild, $sChr, $sPosition)
                  GROUP_CONCAT(DISTINCT t.geneid ORDER BY t.geneid SEPARATOR ";") AS genes,
                  GROUP_CONCAT(DISTINCT
                    IFNULL(i.id,
-                     CONCAT(vog.id, "||", IFNULL(uc.default_license, ""), "||"' .
-            (!$bDNA38? '' : ',
-                       IFNULL(vog.`VariantOnGenome/DNA/hg38`, "")') . ', "||",
-                       vog.effectid, "||"' .
+                     CONCAT(vog.id, "||", IFNULL(uc.default_license, ""), "||", vog.effectid, "||",
+                        IFNULL(vog.`VariantOnGenome/DNA' . (!$this->aActiveGBs[$sBuild]? '' : '/' . $this->aActiveGBs[$sBuild]) . '`, ""), "||"' .
+
```

Comment:
I don't think this is the same functionality. The query used to fetch an alias if it was available. Now it seems to just fetch the selected build's DNA field again.

Reply:
You are right! This was me being a bit clumsy and not removing this while it should have been removed! Later on in the code (read: around line 1110), this big concatenated string is exploded and expects for the item on line 936 to be removed. Because the items after this line were empty in all situations where I tested this, it did not come up!
Thank you for noticing and it has been fixed!